File: backend.NET/Models/CvUpload.cs
using System;
using System.Collections.Generic;

namespace api.Models;

public partial class CvUpload
{
    public string FileId { get; set; } = null!;

    public string StudentId { get; set; } = null!;

    public string? Title { get; set; }

    public string? FileName { get; set; }

    public byte[]? File { get; set; }

    public string? FileType { get; set; }

    public DateTime? UploadedAt { get; set; }

    public virtual Student Student { get; set; } = null!;
}

File: backend/src/models/CVUpload.js
const mongoose = require('mongoose')

const CVUploadSchema = new mongoose.Schema({
  student: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'Student',
    required: true
  },
  title: String,
  fileName: String,
  file: Buffer,
  fileType: {
    type: String,
    default: 'application/pdf'
  },
  uploadedAt: {
    type: Date,
    default: Date.now
  }
}, { timestamps: true })

module.exports = mongoose.model('CVUpload', CVUploadSchema)
